src/osp/Active/SysAreaAssociate.cpp

```diff
@@ -199,8 +199,7 @@ void SysAreaAssociate::floating_origin_translate(Vector3 translation)
 }
 
 
-int SysAreaAssociate::sat_activate(universe::Satellite sat,
-                                   universe::UCompActivatable &satAct)
+int SysAreaAssociate::sat_activate(universe::Satellite sat)
```

Comment:
This looks like it's returning an int with only 3 values, and the rules for understanding what the meaning of the int kind of "assumed".

Better would be to define an enum that clearly indicates what each return value is intended to indicate.

Reply:
No worries, a revamp of SysAreaAssociate is the topic of my next PR.